### PR TITLE
A couple of yq fixups

### PR DESCRIPTION
**Pull request by alpeb**
Fixes linkerd/linkerd2#5801

### CODE REVIEW

**Review by kleimkuhler — APPROVED** [review]

File: linkerd.io/content/2.10/tasks/manually-rotating-control-plane-tls-credentials.md [hunk 1]
```diff
@@ -121,13 +121,8 @@ the new anchor. The following command uses `kubectl` to fetch the Linkerd config
 and `step` to combine it with the newly generated trust anchor:
 
 ```bash
-# For Linkerd < 2.9.0:
```
Comment (lagalbra) [inline linkerd.io/content/2.10/tasks/manually-rotating-control-plane-tls-credentials.md:124]: Why remove the Linkerd < 2.9.0 steps for one doc and not the other? Is Linkerd 2.9.0 the oldest supported version?

File: linkerd.io/content/2.10/tasks/manually-rotating-control-plane-tls-credentials.md [hunk 2]
```diff
@@ -121,13 +121,8 @@ the new anchor. The following command uses `kubectl` to fetch the Linkerd config
...
-kubectl -n linkerd get cm linkerd-config -o=jsonpath='{.data.global}' \
-  | jq -r .identityContext.trustAnchorsPem > original-trust.crt
-
-# For Linkerd >= 2.9.0:
 kubectl -n linkerd get cm linkerd-config -o=jsonpath='{.data.values}' \
-  | yq e .global.identityTrustAnchorsPEM - > original-trust.crt
+  | yq -r .identityTrustAnchorsPEM - > original-trust.crt
```
Comment (lagalbra) [inline linkerd.io/content/2.10/tasks/manually-rotating-control-plane-tls-credentials.md:125]: As noted in https://github.com/linkerd/linkerd2/issues/5801, and changed in #989 , this syntax does not work with yq version 0.4.5

**Review by lagalbra — CHANGES_REQUESTED** [review]

File: linkerd.io/content/2.10/tasks/manually-rotating-control-plane-tls-credentials.md (hunk 1 above)
Comment (alpeb) [reply linkerd.io/content/2.10/tasks/manually-rotating-control-plane-tls-credentials.md:124]: Only as of the release of version `2.10` we started keeping versioned docs. So for versions `2.9` and earlier you should go to https://linkerd.io/2.9/overview/ and for `2.10` the docs are at https://linkerd.io/2.10/overview/
So given there are separate docs for `2.9`, we're not keeping `2.9`-only info in the `2.10` docs.

File: linkerd.io/content/2.10/tasks/manually-rotating-control-plane-tls-credentials.md (hunk 2 above)
Comment (alpeb) [reply linkerd.io/content/2.10/tasks/manually-rotating-control-plane-tls-credentials.md:125]: I'm testing with the `yq` utility from [this repo](https://github.com/kislyuk/yq), whose latest version is `2.12.0`. Can you please point me to what `yq` you're referring to?

Comment (lagalbra) [reply linkerd.io/content/2.10/tasks/manually-rotating-control-plane-tls-credentials.md:125]: I am using version 4.5.0 from [this repo](https://github.com/mikefarah/yq), as that tool (not that particular version though) is referenced in other linkerd docs such as https://linkerd.io/2.10/tasks/installing-multicluster/

Comment (alpeb) [reply linkerd.io/content/2.10/tasks/manually-rotating-control-plane-tls-credentials.md]: Ok good point. I've just pushed a change moving back to `yq e`.

Comment (lagalbra) [reply linkerd.io/content/2.10/tasks/manually-rotating-control-plane-tls-credentials.md]: Across all 3 files, the command should be `yq e .global.identityTrustAnchorsPEM -`; some of the files are missing the last `-` and/or the starting `.global`

Comment (alpeb) [reply linkerd.io/content/2.10/tasks/manually-rotating-control-plane-tls-credentials.md]: Ok I've just added the missing dash. As for `.global`, that disappeared in version `2.10`

**Review by lagalbra — APPROVED** [review]

**Pull request merged**